kubeadm/scripts/PrepareNode.ps1

```diff
@@ -82,10 +81,6 @@ if ($ContainerRuntime -eq "Docker") {
     New-HnsNetwork -Type NAT -Name nat
```

Comment:
We shouldn't create this, it could lead to strange networking issues.

Reply:
removed this